recipe/meta.yaml

```
@@ -61,14 +61,18 @@ outputs:
         - {{ compiler("c") }}
         - {{ compiler("cxx") }}
         - cmake
+        - cuda-driver-dev
+        - git # [linux]
```

FYI, the linter will probably complain once this is merged: you need two spaces before the `# [linux]`.
```suggestion
        - git  # [linux]
```